forgerock-auth/src/main/java/org/forgerock/android/auth/SessionManager.java

```diff
@@ -117,10 +118,28 @@ boolean hasSession() {
      * Close the session, all tokens will be removed.
      */
     public void close() {
+        browserLogout();
         tokenManager.revoke(null);
         singleSignOnManager.revoke(null);
     }
 
+    private void browserLogout() {
```

Comment:
what if they use oauthRedirectUri and use Froptions.discover("")  and ignore the SignOutRedirectUri
 
 would this work

Reply:
i think we are clear here ..resolving this